pkg/cmd/issue/view/view.go

```diff
@@ -114,6 +117,18 @@ func viewRun(opts *ViewOptions) error {
 			lookupFields.Add("comments")
 			lookupFields.Remove("lastComment")
 		}
+
+		// TODO projectsV1Deprecation
+		// Remove this section as we should no longer add projectCards
+		if opts.Detector == nil {
+			cachedClient := api.NewCachedHTTPClient(httpClient, time.Hour*24)
+			opts.Detector = fd.NewDetector(cachedClient, baseRepo.RepoHost())
+		}
+
+		projectsV1Support := opts.Detector.ProjectsV1()
+		if projectsV1Support == gh.ProjectsV1Supported {
```

Comment:
This type might seem a little overkill right now, but in following PRs there are functions that are going to include it in the signature, and the boolean reads very poorly in tests since anonymous bools coerce even if you do `type ProjectV1Support bool`